folsom-semantic-metrics/src/test/java/com/spotify/folsom/client/SemanticFolsomMetricsTest.java

```diff
@@ -93,7 +93,8 @@ public void testMultigetSuccess() {
     metrics.measureMultigetFuture(
         completedFuture(
             ImmutableList.of(
-                GetResult.success(new byte[] {1}, 0), GetResult.success(new byte[] {1}, 0))));
+                GetResult.success(new byte[] {1}, 0, 0),
```

Comment:
If the old success method is re-added, we can keep these tests unchanged I think